main.py

```diff
@@ -587,16 +587,18 @@ def printf(*args, prefix=PLUGIN_NAME):
     print(prefix + ":", *args)
 
 
-def get_project_path(window: sublime.Window) -> 'Optional[str]':
+def get_project_path(window: sublime.Window) -> 'str':
     """
     Returns the common root of all open folders in the window
     """
     if len(window.folders()):
         folder_paths = window.folders()
         return folder_paths[0]
     else:
-        debug("Couldn't determine project directory")
-        return None
+        project_path = os.path.dirname(window.active_view().file_name())
```

Comment:
We might want to guard against View.file_name() returning None (e.g. what if you create a new file and switch syntax to a supported syntax)

Reply:
Ah, indeed. I'll send another commit fixing this.